demo/src/reader.rs

```diff
@@ -129,10 +129,7 @@ impl Reader {
                     }
                 }
                 self.current_tick = Some(t);
-                Ok(Some(RawChunk::Tick {
-                    tick: t,
-                    keyframe: keyframe,
-                }))
+                Ok(Some(RawChunk::Tick { tick: t, keyframe }))
```

Comment:
Where did these changes come from? Is there an automated tool you ran on these files?

Reply:
My IDE marked those as possible improvements, I did those absentmindedly while coding.
I'll simply remove those changes from the commit